app/src/main/java/org/fossasia/phimpme/gallery/activities/SplashScreen.java

```diff
@@ -40,13 +42,13 @@
     public final static int ALBUMS_PREFETCHED = 23;
     public final static int PHOTOS_PREFETCHED = 2;
     public final static int ALBUMS_BACKUP = 60;
-    private boolean PICK_INTENT = false;
+    private static boolean PICK_INTENT = false;
     public final static String ACTION_OPEN_ALBUM = "vn.mbm.phimp.leafpic.OPEN_ALBUM";
 
     //private HandlingAlbums albums;
-    private Album album;
-    private boolean can_be_finished = false;
-    private Intent nextIntent = null;
+    private static Album album;
```

Comment:
It's not initialized and still making it static?

Reply:
I'll make the changes.